src/SFA.DAS.AssessorService.Data/CertificateBatchLogRepository.cs

```diff
@@ -28,6 +31,65 @@ public async Task<CertificateBatchLog> GetCertificateBatchLog(string certificate
             return certificateBatchLog;
         }
 
+        public async Task<List<CertificatePrintSummary>> GetCertificatesForBatch(int batchNumber)
```

Comment:
how come we have moved this method to the `CertificateBatchRepository.cs` I think it made more sense on the `CertificateRepository.cs` ?

Reply:
It operates mainly on the [CertificateBatchLogs] table as it now correctly gets those certificates which were actually included in a given batch rather than those which are currently in a given batch.